Backend/src/modules/users/users.controller.ts

```diff
@@ -1,21 +1,49 @@
-import { type NextFunction, type Request } from 'express';
-import { type users } from '@prisma/client';
-import { HttpStatusCode } from 'axios';
+import { type Response, type NextFunction, type Request } from 'express';
 import UserService from './users.service';
-import { type CustomResponse } from '@/types/common.type';
 import Api from '@/lib/api';
 
 export default class UserController extends Api {
   private readonly userService = new UserService();
 
-  public createUser = async (
+  public authLogin = async (
     req: Request,
-    res: CustomResponse<users>,
+    res: Response,
     next: NextFunction
   ) => {
     try {
-      const user = await this.userService.createUser(req.body);
-      this.send(res, user, HttpStatusCode.Created, 'createUser');
+      res.send('login');
+    } catch (e) {
+      next(e);
+    }
+  };
+
+  public authLogout = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+  ) => {
+    try {
+      // handle with passport
+      req.logout(function (err) {
+        if (err) {
+          console.log(err);
+        }
+        res.redirect('/');
+      });
+    } catch (e) {
+      next(e);
+    }
+  };
+
+  public authRedirectGoogle = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+  ) => {
+    try {
+      // handle with passport
+      res.send(req.user);
+      // res.redirect('/profile');
```

Comment:
```suggestion
```

Reply:
I put this comment to make the profile api later.

Reply:
It is better to not leave comments in the git history, you can put it in your local branch but publish only those code that are actually important.

Reply:
ok, Thank you for telling me that.